dotnet/Directory.Build.props

```diff
@@ -16,13 +16,8 @@
     <IsPackable>disable</IsPackable>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-  </PropertyGroup>
-
-  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
-    <DebugType>portable</DebugType>
+  <PropertyGroup Condition="'$(Configuration)'=='Publish'">
+    <Optimize>True</Optimize>
```

Comment:
Do we need debugsymbols here? Folks just today were asking to use them and very happy they were available on nuget automatically.

Reply:
Maybe we can revive later for `nightly` publishes only.

Reply:
Yeah, we can easily override these msbuild props in the `dotnet build...` command args, so customizing for certain pipelines is no problem. Good idea!